x-pack/legacy/plugins/siem/public/components/open_timeline/delete_timeline_modal/index.tsx

```diff
@@ -4,50 +4,117 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiButtonIcon, EuiModal, EuiToolTip, EuiOverlayMask } from '@elastic/eui';
+import { EuiModal, EuiOverlayMask, EuiIcon, EuiLinkAnchorProps } from '@elastic/eui';
 import React, { useState } from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
 
+import classNames from 'classnames';
 import { DeleteTimelineModal, DELETE_TIMELINE_MODAL_WIDTH } from './delete_timeline_modal';
 import * as i18n from '../translations';
 import { DeleteTimelines } from '../types';
 
+const RemovePopover = createGlobalStyle`
+div.euiPopover__panel-isOpen {
+  display: none;
+}
+`;
+
+type CustomLinkType = EuiLinkAnchorProps & { disabled: boolean };
+
+export const CustomLink = React.forwardRef<HTMLAnchorElement, CustomLinkType>(
```

Comment:
why do we have to create our own link? we can not use `EuiLink`

Reply:
and if we do let's add it in a new file

Reply:
After a zoom chat, we decided to remove the custom link and leave either table or contextMenuItem handles that.